pci_info.py

```diff
@@ -34,107 +35,154 @@
 input_path = "io_input.txt"
 INPUTFILE = configparser.ConfigParser()
 INPUTFILE.optionxform = str
-
 logger = logger_init(filepath=BASE_PATH).getlogger()
 
 
-def create_config(pci_list):
+def create_config_inputs(orig_cfg, new_cfg, inputfile, interface, config_type):
```

Comment:
you do not need to pass old and and new cfg file name.. instead it should fetch default cfg files in fvt wrapper and append the interface name like io_vnic_env3_fvt.cfg to the file also.. to input file io-input.txt .. not sure why you are passing them as parameter